h3/src/stream.rs

```diff
@@ -250,7 +250,7 @@ where
     B: Buf,
 {
     Control(FrameStream<S, B>),
-    Push(u64, FrameStream<S, B>),
```

Comment:
should this warning be suppress? I think this field is needed for Server Push (which is not implemented yet).

Reply:
Can just add the field back when the feature is implemented, IMO.

Reply:
i don't have a strong opinion about this. So ok for me.